payment-request/payment-request-show-method.https.html

```diff
@@ -51,17 +51,16 @@
 
 promise_test(async (t) => {
   const request = new PaymentRequest(defaultMethods, defaultDetails);
-  await test_driver.bless(
-    "test: throws if the promise [[state]] is not 'created'"
-  );
+  await promise_rejects_dom(t, "SecurityError", request.show());
+
+  await test_driver.bless();
   const acceptPromise = request.show(); // Sets state to "interactive"
+
   // No user activation...
-  await promise_rejects_dom(t, "SecurityError", request.show());
+  await promise_rejects_dom(t, "InvalidStateError", request.show());
```

Comment:
This `InvalidStateError` doesn't seem to be related to user activation. Am I missing something?

Reply:
Yeah, that comment needs to go...